src/mmrelay/plugin_loader.py

```
@@ -1386,6 +1386,55 @@ def _try_checkout_and_pull_ref(
+                        repo_path,
+                        GIT_CHECKOUT_CMD,
+                        "-B",
+                        ref_value,
+                        f"{GIT_REMOTE_ORIGIN}/{ref_value}",
+                    ],
+                    timeout=GIT_COMMAND_TIMEOUT_SECONDS,
+                )
+                logger.info(
+                    "Force-synced repository %s to branch %s", repo_name, ref_value
+                )
+                return True
```

Ruff (0.15.7)

[warning] Consider moving this statement to an `else` block

(TRY300)